### DISCUSSION TITLE
Docker images not found in docker hub

### DISCUSSION THREAD

**Discussion by mtmr0x** [microsoft/presidio#1430] [Q&A]
Hey folks, both images (analyzer and anonymizer) are not publicly available at Docker Hub anymore. Any ideas of what happened?

**Comment by mtmr0x** [comment]
an update on this is that the images are showing up at Docker Hub, but I can't pull them

**Reply by omri374** [reply]
Hi @mtmr0x, could you please share how you are pulling the images?

**Reply by mtmr0x** [reply]
Hi @omri374 thanks for answering on this. Sorry for not putting all this info already.

The images were the ones from the [docs](https://microsoft.github.io/presidio/installation/#using-docker) and registry
```
docker pull mcr.microsoft.com/presidio-analyzer
docker pull mcr.microsoft.com/presidio-anonymizer
```

Docker pull in my VM shows nothing and trying to add at Digital Ocean or Render to build VMs from docker images show an error that the image is not public

**Reply by mtmr0x** [reply]
update: downloading at my computer is now working, but I can't pull the image from the DigitalOcean App Platform
